Update faq-oracle-database-azure.md (#124068)
Azure/azure-content: articles/oracle/oracle-db/faq-oracle-database-azure.md

### Do you have any documented benchmark latency-wise between Azure resources and Oracle Database@Azure?

> Latency between Azure resources and Oracle Database@Azure is within the Azure regional latency envelope as the Exadata infrastructure is within the Azure Data Centers. Latency can be further fine-tuned dependent on Co-Location within Availability Zones. For more information, please see [here](https://learn.microsoft.com/azure/reliability/availability-zones-overview?tabs=azure-cli)

Reviewer: ```suggestion
Latency between Azure resources and Oracle Database@Azure is within the Azure regional latency envelope as the Exadata infrastructure is within the Azure Data Centers. Latency can be further fine-tuned dependent on colocation within availability zones. For more information, see [What are availability zones?](/azure/reliability/availability-zones-overview?tabs=azure-cli).
```

In this section, we'll cover questions related to onboarding, provisioning, and migration to Oracle Database@Azure.
### To set up Oracle Database@Azure, what would be the role assignments needed for the Azure user?

> You can find the list of role assignments [here](https://learn.microsoft.com/azure/oracle/oracle-db/oracle-database-groups-roles)

Reviewer: ```suggestion
See [Groups and roles for Oracle Database@Azure](/azure/oracle/oracle-db/oracle-database-groups-roles) for the list of role assignments. 
```

### Can you describe the authentication/authorization standards supported by Oracle Database@Azure?

Oracle Database@Azure is based on SAML and OpenID standards. OCI Oracle Identity and Access Management (IAM) can be federated with Microsoft Entra ID, or other customer used identity providers for OCI Console access for Oracle Database Users.

### Where can I find best practices to plan and deploy Oracle Database@Azure?

> Refer to our landing zone architecture documentation to plan and deploy your oracle workloads with Oracle Database@Azure [here](https://learn.microsoft.com/azure/cloud-adoption-framework/scenarios/oracle-iaas/?wt.mc_id=knwlserapi_inproduct_azportal#landing-zone-architecture-for-oracle-databaseazure)

Reviewer: ```suggestion
To plan and deploy your oracle workloads with Oracle Database@Azure, refer to the [landing zone architecture documentation](/azure/cloud-adoption-framework/scenarios/oracle-iaas/?wt.mc_id=knwlserapi_inproduct_azportal#landing-zone-architecture-for-oracle-databaseazure).
```

### What tools can be used for database migration? Could you help share other details about licensing and charges for these tools?

> There are multiple tools available from Oracle: ZDM, Data Guard, Data pump, GoldenGate, and [more](https://learn.microsoft.com/azure/cloud-adoption-framework/scenarios/oracle-iaas/oracle-migration-planning?wt.mc_id=knwlserapi_inproduct_azportal#migrate-oracle-workloads-to-azure). For more information, contact your Oracle representative for commercials.

Reviewer: ```suggestion
There are multiple tools available from Oracle: ZDM, Data Guard, Data pump, GoldenGate, and more. For more information, see[Migrate Oracle workloads to Azure](https://learn.microsoft.com/azure/cloud-adoption-framework/scenarios/oracle-iaas/oracle-migration-planning?wt.mc_id=knwlserapi_inproduct_azportal#migrate-oracle-workloads-to-azure). Contact your Oracle representative for commercials.
```

> There are multiple tools available from Oracle: ZDM, Data Guard, Data pump, GoldenGate, and more. For more information, see[Migrate Oracle workloads to Azure](https://learn.microsoft.com/azure/cloud-adoption-framework/scenarios/oracle-iaas/oracle-migration-planning?wt.mc_id=knwlserapi_inproduct_azportal#migrate-oracle-workloads-to-azure). Contact your Oracle representative for commercials.

Reviewer: ```suggestion
There are multiple tools available from Oracle: ZDM, Data Guard, Data pump, GoldenGate, and more. For more information, see[Migrate Oracle workloads to Azure](/azure/cloud-adoption-framework/scenarios/oracle-iaas/oracle-migration-planning?wt.mc_id=knwlserapi_inproduct_azportal#migrate-oracle-workloads-to-azure). Contact your Oracle representative for commercials.
```
